src/embrakes/main.cpp

```diff
@@ -58,57 +58,108 @@ void Main::run()
     switch (sm_data_.current_state) {
       case data::State::kIdle:
         if (!tlm_data_.nominal_braking_command) {
-          if (brake_1->checkClamped()) {
-            log_.INFO("Brakes", "Sending retract command");
-            brake_1->sendRetract();
+          if (m_brake->checkClamped()) {
+            m_brake->sendRetract();
+          }
+          if (f_brake->checkClamped()) {
+            f_brake->sendRetract();
           }
+
           Thread::sleep(1000);
-          brake_1->checkHome();
+          m_brake->checkHome();
+          f_brake->checkHome();
+
         } else if (tlm_data_.nominal_braking_command) {
```

Comment:
This seems to be redundant, right? A simple `else` should achieve the same thing.
(I know you didn't actually change anything about this, but I'm reading the code for the first time.)

Reply:
yeah, it's redundant, I'll simplify it!